erpnext/accounts/doctype/asset/asset.js

```diff
@@ -1,299 +1,8 @@
-// Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
+// Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and contributors
 // For license information, please see license.txt
 
-frappe.provide("erpnext.asset");
-
 frappe.ui.form.on('Asset', {
```

Comment:
This is doctype is all deleted ? Why to delete only the code and not the Doctype it self?

Reply:
Because of this validation:
frappe.exceptions.ValidationError: Options must be a valid DocType for field Asset Category in row 18
I have noted it down, all these doctypes will be deleted once this PR is merged.

Reply:
@KanchanChauhan Where are you getting this validation message? I did not get the message.

I have removed all the files from accounts module before merging. Also removed the patch, it was not needed because module will be updated automatically while syncing from json. In your case, it was not happening because you did not delete asset related files from Accounts module.